psi4/src/psi4/libfunctional/LibXCfunctional.cc

```diff
@@ -197,8 +197,8 @@ std::map<std::string, double> LibXCFunctional::query_libxc(const std::string& fu
         double omega, alpha, beta;
         xc_hyb_cam_coef(xc_functional_.get(), &omega, &alpha, &beta);
         params["OMEGA"] = omega;
-        params["ALPHA"] = alpha;
-        params["BETA"] = beta;
+        params["ALPHA"] = alpha + beta;
+        params["BETA"] = std::abs(beta);
```

Comment:
Yes, I think the original implementation of `dict_funcs` was incorrect - otherwise it'd pass the tests. The way I understand our the code is:
```
SR = set_x_alpha              =  LibXC_ALPHA + LibXC_BETA
LR = set_x_alpha + set_x_beta = (LibXC_ALPHA + LibXC_BETA) + (- LibXC_BETA) = LibXC_ALPHA
```
This means the `abs(beta)` should be `-1 * beta` as there are functionals for which LR is smaller than SR (ie `LibXC_BETA > 0`).

Reply:
I agree, like I commented below.

Reply:
Gotcha, fixed.